src/query.js

```diff
@@ -1024,6 +1024,23 @@ function CreateRole(params) {
   return new Expr({ create_role: wrap(params) })
 }
 
+/**
+ * See the [docs](https://app.fauna.com/documentation/reference/queryapi#write-functions).
+ *
+ * @param {module:query~ExprArg} params
+ *   An object of parameters used to create a new access provider.
+ *     - name: A valid schema name
+ *     - issuer: A unique string
+ *     - jwks_uri: A valid HTTPS URI
```

Comment:
@nathaniel-may @marrony The JIRA ticket referred to this field as `jwks_url`, but Core has it as `jwks_uri`. Let me know if you want to change it back for any reason 👍